index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
```

Comment:
If you like Code Academy, check out their course on html and CSS: https://www.codecademy.com/learn/web

As the summer progresses, you're going to get to a point where those simple tutorials won't be in-depth enough and you'll need to talk to me another another dev here, but until you feel solid with the basics, I think it's great if you supplement what we're doing here with online courses.